examples/helm/tiller-template.yaml

```diff
@@ -39,6 +40,20 @@ objects:
   - kind: ServiceAccount
     name: tiller
 
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: RoleBinding
+  metadata:
+    name: edit-79929
+    namespace: ${TILLER_NAMESPACE}
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: edit
```

Comment:
can you not add whatever additional permissions you need to the existing "tiller" role being created above?

Reply:
Yes it's better. I will fix it soon as possible.